### ISSUE TITLE
[Blue Burst] Error Client Command: Can't overwrite existing player with NPC while playing Seat of Heart

### ISSUE THREAD

**Issue by nolrinale** [fuzziqersoftware/newserv#294]
NOTE: To play this quest you must first clear Gov Quest `7-5 Isle of Mutants`

Create a new Episode 2 solo room and begin `Seat of Heart`, play through the VR Temple area, there will be a point ELLY will walk towards the 2nd console to hack it to let you proceed but before that you will get an Error 100 with the error in console:

```
W 19397 2023-12-27 17:26:08 - [Server] Error processing client command: cannot overwrite existing player with NPC
```

Full Log up to that Point, impossible to proceed further
[SeatOfHeartNPCReplaceError.txt](https://github.com/fuzziqersoftware/newserv/files/13779909/SeatOfHeartNPCReplaceError.txt)

**Comment by fuzziqersoftware** [comment]
This should be fixed by 9efdf881.

**Comment by nolrinale** [comment]
Testing with the latest commit the same error happens now right after starting the quest when ELLY joins your party for the very first time in Pioneer2

```
I 971 2023-12-28 15:06:56 - [Commands] Sending to C-38 (TOUKA) (version=BB command=00DE flag=00000000) 
0000 | 28 00 DE 00 00 00 00 00 FF FF FF FF FF FF FF FF | ( 
0010 | FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF | 
0020 | FF FF FF FF FF FF FF FF | 
I 971 2023-12-28 15:06:56 - [Commands] Received from C-38 (TOUKA) (version=BB command=0013 flag=00000047) 
0000 | 18 00 13 00 47 00 00 00 71 75 65 73 74 32 37 2E | G quest27. 
0010 | 62 69 6E 00 00 00 00 00 | bin 
I 971 2023-12-28 15:06:56 - [C-38] Done sending file quest27.bin 
I 971 2023-12-28 15:06:56 - [Commands] Received from C-38 (TOUKA) (version=BB command=0060 flag=00000000) 
0000 | 14 00 60 00 00 00 00 00 52 03 00 00 08 00 00 00 | ` R 
0010 | 50 B9 FF FF | P 
I 971 2023-12-28 15:06:56 - [Commands] Received from C-38 (TOUKA) (version=BB command=0060 flag=00000000) 
0000 | 0C 00 60 00 00 00 00 00 22 01 00 00 | ` " 
I 971 2023-12-28 15:06:59 - [Commands] Received from C-38 (TOUKA) (version=BB command=0060 flag=00000000) 
0000 | 20 00 60 00 00 00 00 00 3F 06 00 00 00 00 00 C0 | ` ? 
0010 | 00 00 00 00 00 00 F0 C2 00 00 00 00 00 00 20 43 | C 
I 971 2023-12-28 15:06:59 - [Commands] Received from C-38 (TOUKA) (version=BB command=0060 flag=00000000) 
0000 | 10 00 60 00 00 00 00 00 1F 02 00 00 00 00 00 00 | ` 
I 971 2023-12-28 15:06:59 - [Commands] Received from C-38 (TOUKA) (version=BB command=0060 flag=00000000) 
0000 | 0C 00 60 00 00 00 00 00 3B 01 00 00 | ` ; 
I 971 2023-12-28 15:06:59 - [Commands] Received from C-38 (TOUKA) (version=BB command=0060 flag=00000000) 
0000 | 0C 00 60 00 00 00 00 00 23 01 00 00 | ` # 
I 971 2023-12-28 15:06:59 - [Commands] Received from C-38 (TOUKA) (version=BB command=016F flag=00000000) 
0000 | 08 00 6F 01 00 00 00 00 | o 
I 971 2023-12-28 15:06:59 - [Commands] Sending to C-38 (TOUKA) (version=BB command=00B1 flag=00000000) 
0000 | 20 00 B1 00 00 00 00 00 32 30 32 33 3A 31 32 3A | 2023:12: 
0010 | 32 38 3A 20 31 35 3A 30 36 3A 35 39 2E 30 30 30 | 28: 15:06:59.000 
I 971 2023-12-28 15:06:59 - [Commands] Sending to C-38 (TOUKA) (version=BB command=0060 flag=00000000) 
0000 | 0C 00 60 00 00 00 00 00 DD 01 01 00 | ` 
I 971 2023-12-28 15:06:59 - [Commands] Sending to C-38 (TOUKA) (version=BB command=1DEA flag=00000000) 
0000 | 08 00 EA 1D 00 00 00 00 | 
I 971 2023-12-28 15:06:59 - [Commands] Sending to C-38 (TOUKA) (version=BB command=15EA flag=00000001) 
0000 | 64 08 EA 15 01 00 00 00 DD D8 37 62 04 00 00 00 | d 7b 
0010 | 00 00 00 00 00 00 00 00 40 00 00 00 09 00 45 00 | @ E 
0020 | 4B 00 49 00 59 00 4F 00 4D 00 49 00 5A 00 55 00 | K I Y O M I Z U 
0030 | 44 00 45 00 52 00 41 00 00 00 00 00 DD D8 37 62 | D E R A 7b 
0040 | 00 00 00 00 09 00 45 00 54 00 4F 00 55 00 4B 00 | E T O U K 
0050 | 41 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | A 
0060 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0070 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0080 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0090 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
00A0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
00B0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
00C0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
00D0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
00E0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
00F0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0100 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0110 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0120 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0130 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0140 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0150 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0160 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0170 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0180 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0190 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
01A0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
01B0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
01C0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
01D0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
01E0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
01F0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0200 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0210 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0220 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0230 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0240 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0250 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0260 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0270 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0280 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0290 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
02A0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
02B0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
02C0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
02D0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
02E0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
02F0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0300 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0310 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0320 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0330 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0340 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0350 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0360 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0370 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0380 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0390 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
03A0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
03B0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
03C0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
03D0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
03E0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
03F0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0400 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0410 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0420 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0430 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0440 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0450 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0460 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0470 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0480 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0490 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
04A0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
04B0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
04C0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
04D0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
04E0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
04F0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0500 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0510 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0520 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0530 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0540 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0550 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0560 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0570 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0580 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0590 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
05A0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
05B0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
05C0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
05D0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
05E0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
05F0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0600 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0610 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0620 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0630 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0640 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0650 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0660 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0670 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0680 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0690 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
06A0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
06B0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
06C0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
06D0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
06E0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
06F0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0700 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0710 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0720 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0730 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0740 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0750 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0760 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0770 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0780 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0790 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
07A0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
07B0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
07C0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
07D0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
07E0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
07F0 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0800 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0810 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0820 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0830 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0840 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0850 | 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | 
0860 | 00 00 00 00 | 
I 971 2023-12-28 15:06:59 - [Commands] Sending to C-38 (TOUKA) (version=BB command=00DE flag=00000000) 
0000 | 28 00 DE 00 00 00 00 00 FF FF FF FF FF FF FF FF | ( 
0010 | FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF | 
0020 | FF FF FF FF FF FF FF FF | 
I 971 2023-12-28 15:07:00 - [Commands] Received from C-38 (TOUKA) (version=BB command=0060 flag=00000000) 
0000 | 14 00 60 00 00 00 00 00 69 03 85 00 00 00 01 00 | ` i 
0010 | 00 00 04 00 | 
I 971 2023-12-28 15:07:00 - [Commands] Received from C-38 (TOUKA) (version=BB command=0060 flag=00000000) 
0000 | 20 00 60 00 00 00 00 00 3F 06 01 00 00 00 00 00 | ` ? 
0010 | 0F 00 00 00 00 C0 8F 44 00 00 00 00 00 00 48 44 | D HD 
I 971 2023-12-28 15:07:00 - [Commands] Received from C-38 (TOUKA) (version=BB command=0060 flag=00000000) 
0000 | 1C 00 60 00 00 00 00 00 24 05 01 00 00 00 00 00 | ` $ 
0010 | 00 C0 8F 44 00 00 00 00 00 00 48 44 | D HD 
I 971 2023-12-28 15:07:00 - [Commands] Received from C-38 (TOUKA) (version=BB command=0060 flag=00000000) 
0000 | 14 00 60 00 00 00 00 00 69 03 79 01 01 00 6B 00 | ` i y k 
0010 | 01 00 00 00 | 
I 971 2023-12-28 15:07:00 - [Commands] Received from C-38 (TOUKA) (version=BB command=0060 flag=00000000) 
0000 | 14 00 60 00 00 00 00 00 26 03 01 00 00 00 21 00 | ` & ! 
0010 | 00 00 00 00 | 
I 971 2023-12-28 15:07:00 - [Commands] Received from C-38 (TOUKA) (version=BB command=0060 flag=00000000) 
0000 | 14 00 60 00 00 00 00 00 26 03 01 00 01 00 21 00 | ` & ! 
0010 | 00 00 00 00 | 
I 971 2023-12-28 15:07:00 - [Commands] Received from C-38 (TOUKA) (version=BB command=0060 flag=00000000) 
0000 | 14 00 60 00 00 00 00 00 26 03 01 00 02 00 21 00 | ` & ! 
0010 | 00 00 00 00 | 
I 971 2023-12-28 15:07:00 - [Commands] Received from C-38 (TOUKA) (version=BB command=0060 flag=00000000) 
0000 | 24 00 60 00 00 00 00 00 2B 07 01 00 00 06 03 00 | $ ` + 
0010 | 00 00 00 00 00 00 00 00 19 00 01 00 00 00 00 00 | 
0020 | 02 00 00 00 | 
I 971 2023-12-28 15:07:00 - [Commands] Received from C-38 (TOUKA) (version=BB command=0060 flag=00000000) 
0000 | 14 00 60 00 00 00 00 00 25 03 01 00 19 00 01 00 | ` % 
0010 | 00 00 00 00 | 
I 971 2023-12-28 15:07:00 - [Commands] Received from C-38 (TOUKA) (version=BB command=0060 flag=00000000) 
0000 | 24 00 60 00 00 00 00 00 2B 07 01 00 01 01 16 00 | $ ` + 
0010 | 00 00 00 00 00 00 00 00 1A 00 01 00 00 00 00 00 | 
0020 | 02 00 00 00 | 
I 971 2023-12-28 15:07:00 - [Commands] Received from C-38 (TOUKA) (version=BB command=0060 flag=00000000) 
0000 | 14 00 60 00 00 00 00 00 25 03 01 00 1A 00 01 00 | ` % 
0010 | 00 00 00 00 | 
I 971 2023-12-28 15:07:00 - [Commands] Received from C-38 (TOUKA) (version=BB command=0060 flag=00000000) 
0000 | 24 00 60 00 00 00 00 00 2B 07 01 00 01 02 19 00 | $ ` + 
0010 | 00 00 00 00 00 00 00 00 1B 00 01 00 00 00 00 00 | 
0020 | 02 00 00 00 | 
I 971 2023-12-28 15:07:00 - [Commands] Received from C-38 (TOUKA) (version=BB command=0060 flag=00000000) 
0000 | 14 00 60 00 00 00 00 00 25 03 01 00 1B 00 01 00 | ` % 
0010 | 00 00 00 00 | 
I 971 2023-12-28 15:07:00 - [Commands] Received from C-38 (TOUKA) (version=BB command=0060 flag=00000000) 
0000 | 1C 00 60 00 00 00 00 00 24 05 00 00 00 C0 FF FF | ` $ 
0010 | 00 00 F0 C2 00 00 00 00 00 00 20 43 | C 
I 971 2023-12-28 15:07:00 - [Commands] Received from C-38 (TOUKA) (version=BB command=0060 flag=00000000) 
0000 | 1C 00 60 00 00 00 00 00 24 05 01 00 00 40 00 00 | ` $ @ 
0010 | 00 00 0C C3 00 00 00 00 00 00 20 43 | C 
I 971 2023-12-28 15:07:13 - Saved system file system/players/system_myria.psosys 
I 971 2023-12-28 15:07:13 - Added 59 seconds to play time 
I 971 2023-12-28 15:07:13 - Saved character file system/players/player_myria_1.psochar 
I 971 2023-12-28 15:07:13 - Saved Guild Card file system/players/guild_cards_myria.psocard 
I 971 2023-12-28 15:07:15 - [Commands] Received from C-38 (TOUKA) (version=BB command=0060 flag=00000000) 
0000 | 14 00 60 00 00 00 00 00 69 03 00 00 00 00 CD 02 | ` i 
0010 | 02 00 6B 00 | k 
W 971 2023-12-28 15:07:15 - [Server] Error processing client command: cannot create or modify NPC in existing player slot 
I 971 2023-12-28 15:07:15 - [Server] Client disconnected: C-38 on fd 37 
I 971 2023-12-28 15:07:15 - [Lobby:00000018:FloorItems:00] Deleted 0 private items 
I 971 2023-12-28 15:07:15 - [Lobby:00000018:FloorItems:01] Deleted 0 private items 
I 971 2023-12-28 15:07:15 - [Lobby:00000018:FloorItems:02] Deleted 0 private items 
I 971 2023-12-28 15:07:15 - [Lobby:00000018:FloorItems:03] Deleted 0 private items 
I 971 2023-12-28 15:07:15 - [Lobby:00000018:FloorItems:04] Deleted 0 private items 
I 971 2023-12-28 15:07:15 - [Lobby:00000018:FloorItems:05] Deleted 0 private items 
I 971 2023-12-28 15:07:15 - [Lobby:00000018:FloorItems:06] Deleted 0 private items 
I 971 2023-12-28 15:07:15 - [Lobby:00000018:FloorItems:07] Deleted 0 private items 
I 971 2023-12-28 15:07:15 - [Lobby:00000018:FloorItems:08] Deleted 0 private items 
I 971 2023-12-28 15:07:15 - [Lobby:00000018:FloorItems:09] Deleted 0 private items 
I 971 2023-12-28 15:07:15 - [Lobby:00000018:FloorItems:0A] Deleted 0 private items 
I 971 2023-12-28 15:07:15 - [Lobby:00000018:FloorItems:0B] Deleted 0 private items 
I 971 2023-12-28 15:07:15 - [Lobby:00000018:FloorItems:0C] Deleted 0 private items 
I 971 2023-12-28 15:07:15 - [Lobby:00000018:FloorItems:0D] Deleted 0 private items 
I 971 2023-12-28 15:07:15 - [Lobby:00000018:FloorItems:0E] Deleted 0 private items 
I 971 2023-12-28 15:07:15 - [Lobby:00000018:FloorItems:0F] Deleted 0 private items 
I 971 2023-12-28 15:07:15 - [Lobby:00000018:FloorItems:10] Deleted 0 private items 
I 971 2023-12-28 15:07:15 - [Lobby:00000018:FloorItems:11] Deleted 0 private items 
I 971 2023-12-28 15:07:15 - [Lobby:18] Enqueued for deletion 
I 971 2023-12-28 15:07:15 - [Lobby:18] Deleted 
I 971 2023-12-28 15:07:15 - Saved system file system/players/system_myria.psosys 
I 971 2023-12-28 15:07:15 - Added 2 seconds to play time 


```

**Comment by fuzziqersoftware** [comment]
I'll take another look soon.

**Comment by fuzziqersoftware** [comment]
This should be fixed by b6cfb5b2 - it turns out subcommand 6x69x02 doesn't actually have a client ID in it.
